### ISSUE TITLE
Retirement of /healthbenefits/ area of legacy va.gov

### ISSUE THREAD

**Issue by mnorthuis** [department-of-veterans-affairs/va.gov-team#1506]
The goal is to retire or move all content that currently lives under va.gov/healthbenefits/.  This area is the legacy health benefits content.  Most of the content has been or is in the process of being rewritten and migrated to the Health Benefits hub.   There is some content that is not Veteran facing and will need a new home. 

- Update the Legacy Left Nav document sent to VHA stakeholders based on what content has officially been   rewritten and launched; include the new URL
- Complete all legacy content retirement and implement redirects as planned
- Audit and document all remaining content under /healthbenefits/ that has not been rewritten/migrated - will need assistance from VHA stakeholders
- Determine placement for remaining content, and identify if any additional content needs to be rewritten/migrated
- Provide recommendations for revised legacy/Team Site menu for VHA content that accomodates /health as well as all applicable sub-domains.

**Comment by Karak888** [comment]
@Karak888 to cross check against other Epic.

**Comment by mnorthuis** [comment]
All current issues in this epic are complete. Leaving epic open until feedback is received from stakeholders on recommendations and next steps are determined. 

[LegacyLeftNavRecs-VHA-10182019.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/3745232/LegacyLeftNavRecs-VHA-10182019.docx)

**Comment by Karak888** [comment]
@AnneHurley @jenniferlee-dsva  has this been reviewed by stakeholders yet? Thanks!

**Comment by Karak888** [comment]
@mnorthuis I spoke with @jenniferlee-dsva yesterday and she still needs to run this by JHL and believe she has just gotten back from being out for a bit. Thanks!!!

**Comment by Karak888** [comment]
waiting on @jenny-h-l  JHL and VHA to review spreadsheet.

**Comment by aeltarhoni** [comment]
@mnorthuis Is there anything else we need to do for client side redirects?

**Comment by mnorthuis** [comment]
All work on this issue has been completed, waiting for JHL to provide feedback.  Closing this issue, can open new issue for any future work.
